evm.md

```diff
@@ -283,8 +283,8 @@ OpCode Execution
 ```k
     syntax KItem ::= "#execute"
  // ---------------------------
-    rule <k> (. => #next) ~> #execute ... </k>
-    rule <k> #halt ~> (#execute => .) ... </k>
+    rule <k> (. => #next) ~> #execute ... </k> [tag(step)]
+    rule <k> #halt ~> (#execute => .) ... </k> [tag(step)]
```

Comment:
This should probably be a rule name/label instead of a rule tag, ie:

```
rule [step] : <k> (. => #next) ~> #execution ... </k>
rule [halt] : <k> #halt ~> (#execute => .) ... </k>
```

Note that rule labels must be unique. The reason I suggest a rule label instead of a `tag` is because using `tag` may mess up the strategy used for execution.

Reply:
Note that rule labels/names are passed through as attributes too. So if you're just searching for the `step` and `halt` string as a key in the attributes to detect this, it should still work.

Reply:
@ehildenb tags will not affect this definition at all because it doesn't import any strategy module.